### DISCUSSION TITLE
How to launch stellarium on a secondary screen?

### DISCUSSION THREAD

**Discussion by Oursblanc067** [Stellarium/stellarium#155] [Q&A] [answered]
Before reporting an issue, please *absolutely make sure* to check the closed issues 
(https://github.com/Stellarium/stellarium/issues?q=is%3Aissue+is%3Aclosed) 
whether it has been reported and solved/closed already! Don't report in this case.

Also look into the wiki (https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version) 
and check if you are attempting to report a known issue. Don't report in this case.

Also look into the User Guide before reporting unexpected but correct behaviour. Don't report in this case.

If you want to report a feature wish, delete these template lines above and below, and fill in. 

If you want to report a bug, please make sure to update and verify the bug still exists in the current version. 
In this case please also check current beta builds from http://stellarium.org
If bug still persists, delete text above down to and including this line and fill in:

### Expected Behaviour
Unable to move Stellarium Windows to another screens attached to the Laptop.

### Actual Behaviour
Stellarium starts on the default screen (Laptop)

### Steps to reproduce
Have 2 screens. Launch Stellarium. Unable to transfert Stellarium windows on the other screen

### System
* Stellarium version: <0.18.0>
* Operating system: <Windows 10, 1803>
* Graphics Card: <Intel HD Graphics 4000), Driver 9.17.10.4229>

### Logfile 
N/A

**Comment by gzotti** [comment, marked as answer]
Seriously? 
- F11 to leave fullscreen
- Drag window to other screen
- F11 to enter fullscreen on other screen

**Comment by Oursblanc067** [comment]
Sorry.. Did’nt see ☹

 

De : Georg Zotti <notifications@github.com> 
Envoyé : lundi 14 mai 2018 17:14
À : Stellarium/stellarium <stellarium@noreply.github.com>
Cc : Oursblanc067 <photos@oursblanc.net>; Author <author@noreply.github.com>
Objet : Re: [Stellarium/stellarium] How to launch stellarium on a secondary screen? (#155)

 

Seriously?

*	F11 to leave fullscreen
*	Drag window to other screen
*	F11 to enter fullscreen on other screen
